src/data/model/LayerTreeNode.js

```diff
@@ -115,6 +115,10 @@ Ext.define('GeoExt.data.model.LayerTreeNode', {
 
         // forward changes to ol object
         if (key === 'checked') {
+            if (me.data.checked === newValue && !classicMode) {
```

Comment:
Is the check `me.data.checked === newValue` necessary? Should be always the case since this is done by the `me.callParent(arguments);`. But should not hurt to check.

Reply:
I'm not really sure right now, this function gets called A LOT (at least when using classic mode).

Reply:
Would it make sense to turn the condition around then? 

```js
if (!classicMode && me.data.checked === newValue) {
```

Reply:
Done in 6b5597ee